test/e2e_node/oomkiller_linux_test.go

```diff
@@ -37,6 +38,26 @@ type testCase struct {
 	oomTargetContainerName string
 }
 
+const KubeReservedMemory = "1500Mi"
```

Comment:
We don't really know the exact machine this test will run on, so I would avoid hardcoding it to a specific number. Can we make it a percentage of the node's capacity?

Reply:
PTAL